src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -0,0 +1,126 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="documentation-hero" :style="styles">
+    <NavigatorLeafIcon :type="type" key="first" class="background-icon first-icon" with-colors />
```

Comment:
I'm seeing some unexpected results in terms of mapping icons to page types, but that can be addressed in a separate PR since it wouldn't impact any of the changes here.

Reply:
Yeah there are. @ethan-kusters told me that they are trying to unify them. We should make a list of inconsistencies and verify with the docc and design team.